src/main/java/com/dnastack/ga4gh/search/Ga4ghSearchAdapterPrestoApplication.java

```diff
@@ -1,12 +1,14 @@
 package com.dnastack.ga4gh.search;
 
+import com.dnastack.audit.config.AuditEventLoggerProperties;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
-import org.springframework.context.annotation.ComponentScan;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
 import org.springframework.scheduling.annotation.EnableScheduling;
 
 @EnableScheduling
 @SpringBootApplication
+@EnableConfigurationProperties(AuditEventLoggerProperties.class) // can be removed on spring-boot version upgrade
```

Comment:
I'm glad you put this comment here. I hadn't previously realized why some of our apps needed this and others didn't.